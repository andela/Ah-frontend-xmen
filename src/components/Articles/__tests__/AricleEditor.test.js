import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import Editor from '../ArticleEditor';

describe('<Editor />', () => {
  const props = {
    onChange: jest.fn(),
    onSubmit: jest.fn(),
    onInputChange: jest.fn(),
    onBodyChange: jest.fn(),
    title: 'title',
    description: 'description',
    body: 'body',
    tags: ['tags'],
  };
  it('should render wihtout crashing', () => {
    const wrapper = shallow(<Editor {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
