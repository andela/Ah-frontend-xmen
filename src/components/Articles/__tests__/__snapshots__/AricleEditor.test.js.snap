// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Editor /> should render wihtout crashing 1`] = `ShallowWrapper {}`;
